### ISSUE TITLE
[SUGGESTION]: Adding the event pokémon from Pokémon GO to Pokéclicker

### ISSUE THREAD

**Issue by Bedeku** [pokeclicker/pokeclicker#4986]
### Current Behavior

Among the different pokémon variants that have been added to Pokéclicker, I've noticed a distinct lack of the many variants that were introduced in Pokémon GO.

### Improved Behavior

It would help increase the list of event pokémon that people can hunt during specific dates.

### Reason for change

They're easier additions, given that they're pre-existent.

### Detailed Description

Example for winter-related events:
![image](https://github.com/pokeclicker/pokeclicker/assets/115491024/59372b51-fb0b-47ac-b92b-2f2279c27e4c)
![image](https://github.com/pokeclicker/pokeclicker/assets/115491024/b6649ea0-aa9f-4c96-9c8a-15857337a675)

Example for spring-related events:
![image](https://github.com/pokeclicker/pokeclicker/assets/115491024/009cb231-1989-4f7a-9e88-7412bda8b0c8)

Example for halloween events:
![image](https://github.com/pokeclicker/pokeclicker/assets/115491024/813307a7-8ba0-4e71-9bc9-4ee6be7799cc)
![image](https://github.com/pokeclicker/pokeclicker/assets/115491024/eec29496-5b70-4893-8129-f708698b28f2)

**Comment by Someonealive-QN** [comment]
You can notice that most of those are basically pokémons with a themed hat which we don't think is a good enough change for a form based on that to be added (some forms are an exception such as flowering togepi which was used to swap out surprise togepi)

**Comment by milotic** [comment]
can work with Pokes of full costume, all have official plush

**Comment by Someonealive-QN** [comment]
That's true but the problem is implementation, just adding it to the code and boom done is not easy, since you need approval of the devs for that specific pokémon to be added and then think of a way of adding it that's unique (during the xmas event you can notice none of the pokémons are obtained by the same method, all of them are unique)

**Comment by Someonealive-QN** [comment]
By the same method i mean as like they both don't share the same way to obtain (reindeer stantler is only in johto while santa snorlax is in all regions so they are obtained a little differently)
